### DISCUSSION TITLE
psr/container v2 support

### DISCUSSION THREAD

**Discussion by alex-winter** [composer-unused/composer-unused#237] [Q&A] [answered]
I would like to use this software on a project however when I try to install it locally to the project, it conflicts with the psr/container version the project supports (^2) 

Would it be possible to add support for psr/container ^2?

alternatively maybe there is another solution I am overlooking which does not require composer-unused to be changed at all

**Comment by icanhazstring** [comment, marked as answer]
Hi,

I think there is nothing speaking against supporting v2 of psr/container.

You could however also use the phar itself which is bundled with its own dependencies so you don't need to worry about conflicting dependencies.

But, you could also open a PR adding support for psr/container v2. If the build is working, nothing should break 👍
